File: CourseProjects/CourseProjects/VisualStudioProjects/Languages/Basic VB Programming/Properties/Tester.vb
Module Tester

    Sub Main()
        Dim cust As New Customer("Joe Bloggs")
        Console.WriteLine("Customers name is: {0}", cust.Name)
        'Will not compile - Name is a Read Only Property
        'cust.Name = "Dave Bloggs"
        cust.Address = "10 Arcatia Road"
        Console.WriteLine("Just set address to {0}", cust.Address)
        cust.CreditLimit = 30000
        'Will not compile - CreditLimit is a Write Only Property
        'Dim tmp As Double = cust.CreditLimit
        Console.WriteLine("Final details are: {0}", cust)
    End Sub

End Module

File: CourseProjects/CourseProjects/VisualStudioProjects/Languages/Basic CS  Programming/Properties/Tester.cs
using System;

namespace CSharpProperties {
	class Tester {
		static void Main(string[] args) {
			Person person = new Person();
			person.Name = "Joe Bloggs";
			person.Age = 27;
			person.Salary = 30000.00;

			Console.WriteLine("Person called {0} of age {1}, earning {2}",person.Name,person.Age, person.Salary);
		}
	}
}